src/dynamicResources/model/supported_resources.json

```diff
@@ -1796,10 +1796,10 @@
         "documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lookoutvision-project.html"
     },
```

Comment:
no idea why I include changes on this file too

Reply:
seems I already merged from mainline here https://github.com/aws/aws-toolkit-vscode/blob/master/src/dynamicResources/model/supported_resources.json#L389, but diff here doesn't work

Reply:
I think it's just `prettier`